lib/mayaUsd/render/vp2RenderDelegate/mayaPrimCommon.h

```diff
@@ -258,7 +260,17 @@ class MayaUsdRPrim
         ReprVector const&  reprs,
         TfToken const&     renderTag);
 
-    void _SyncDisplayLayerModes(const HdRprim& refThis);
+    void _SyncDisplayLayerModes();
+    void _SyncDisplayLayerModesInstanced(SdfPath const& id, unsigned int instanceCount);
+
+    bool _ShouldSkipInstance(unsigned int usdInstanceId, const TfToken& reprToken) const;
+
+    void _SyncForcedReprs(
+        HdRprim&          refThis,
+        HdSceneDelegate*  delegate,
+        HdRenderParam*    renderParam,
+        HdDirtyBits*      dirtyBits,
+        ReprVector const& reprs);
```

Comment:
I'd like to have some docs/comment that explain what each function does and why. I guess it falls into a larger scope of documenting how the render delegates work.